### ISSUE TITLE
[Feature Enhancement] Config for default Pneumatic Armor color values

### ISSUE THREAD

**Issue by HipHopHuman** [TeamPneumatic/pnc-repressurized#951]
Provide client configuration options to set values that the Pneumatic Armour colours default to. 

Context:

- It will allow packmakers to add a bit of polish to their packs
- I noticed that the visor color setting affects the color of the HUD
- I typically configure Xaero's Minimap to be circular with a transparent Aqua Frame
- I like to match this frame with the Pneumatic Helmet HUD by making it Aqua as well, because it makes Xaero's seem like it's part of the HUD
- I figured this would be really cool if a freshly installed modpack could tweak these values to have it work like this out of the box

I am totally OK with a default colour config for the visor piece only, if you're concerned with keeping the default actual armor color theme for (branding...?) purposes

**Comment by desht** [comment]
Nah, it can all be configurable.  The default is just a neutral grey colour with no real significance attached to it.
